lib/improver/tests/blending/weighted_blend/test_unify_forecast_reference_time.py

```diff
@@ -44,6 +45,32 @@
 from improver.blending.weighted_blend import unify_forecast_reference_time
 
 
+def convert_time_to_seconds(cube):
```

Comment:
This is a useful function and shouldn't be in a unit-test file but in improver.utilities.temporal where it can be unit-tested itself. We will want this for #716. I'm told that @cgsandford is writing another PR about cf-time, but I cannot find the ticket.